decentraland/unity-renderer
# Feat: Infinite Floor

## What does this PR change?



This PR adds the new infinite floor to the world.

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch=feat/infinite-floor
2. Move around the world and check the infinite floor is being rendered properly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
